fanficfare/adapters/adapter_inkbunnynet.py

```diff
@@ -203,7 +203,7 @@ def getChapterText(self, url):
 
     def before_get_urls_from_page(self,url,normalize):
         # To display the links to stories that are not available to guests.
-        if self.getConfig("username") and self.getConfig("always_login"):
```

Comment:
Actually, something very similar came up recently:
https://github.com/JimmXinu/FanFicFare/issues/1135#issuecomment-2543996159

It would be better to keep `always_login` and add the comments to defaults.ini than remove it.